e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/company-details-continue-no-errors.spec.js

```diff
@@ -0,0 +1,56 @@
+import { companyDetails } from '../../../../pages/your-business';
+import { submitButton, yesRadioInput } from '../../../../pages/shared';
+import {
+  ROUTES, VALID_PHONE_NUMBERS, WEBSITE_EXAMPLES, COMPANIES_HOUSE_NUMBER,
+} from '../../../../../../constants';
+import getReferenceNumber from '../../../../helpers/get-reference-number';
+
+const natureOfBusinessUrl = `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.EXPORTER_BUSINESS.NATURE_OF_BUSINESS}`;
+
+describe("Insurance - Your business - Company details page - As an Exporter I want to enter details about my business in 'your business' section", () => {
+  let referenceNumber;
+  let url;
+
+  before(() => {
+    cy.navigateToUrl(ROUTES.INSURANCE.START);
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    getReferenceNumber().then((id) => {
+      referenceNumber = id;
+
+      url = `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ROUTES.INSURANCE.EXPORTER_BUSINESS.COMPANY_DETAILS}`;
```

Comment:
Could destructure both the URLs here (line 8 as well), not a blocker

Reply:
it refuses to work because of the reference number